### ISSUE TITLE
AttributeError: 'DecodedSamples' object has no attribute 'to'

### ISSUE THREAD

**Issue by m17reardon** [Engineer-of-Stuff/stable-diffusion-paperspace#109]
Been getting this error all day after the automatic1111 update. Has anyone found a fix. Or experienced the same issue?

**Comment by Cyberes** [comment]
Yeah, I don't recommend updating the WebUI unless you need a new feature. Unless you like spending a couple hours debugging and reading GitHub issues. Honestly, I suggest reverting to whatever commit worked.

**Comment by Kyuuto312** [comment]
How do I revert to a previous commit? I've also updated without realizing and want to revert back.

**Comment by Cyberes** [comment]
@Kyuuto312 
1. Back up your current install: `cp -r stable-diffusion-webui stable-diffusion-webui-latest`
2. Pick a commit from https://github.com/AUTOMATIC1111/stable-diffusion-webui/commits/master
3. `git reset --hard [commit]`
